[synthetics_test] Add assertRequests browser step (#2971)
DataDog/terraform-provider-datadog: docs/resources/synthetics_test.md

- `modifiers` (List of String) Modifier to use for a "press key" step.
- `playing_tab_id` (String) ID of the tab to play the subtest.
- `request` (String) Request for an API step.

> - `requests` (String) Details of the requests for an "assert request" step, JSON encoded string. Refer to the examples for a usage example showing the schema.

Reviewer: same :D

```suggestion
- `requests` (String) Details of the requests for an "assert request" step. It is a JSON encoded string. Refer to the schema for usage examples.
```
